### DISCUSSION TITLE
Something curious about euphony logo

### DISCUSSION THREAD

**Discussion by happy-refactoring** [euphony-io/euphony#131] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/108911952/178092361-3ae15b64-5841-434e-89ab-cf0696411c16.png)

The euphony logo looks like a pinwheel.
The pinwheel looks like it's made up four things.

What is a pinwhell made of?

**Comment by serim53** [comment]
음표

**Comment by zion830** [comment]
손가락

**Comment by serim53** [comment]
낮은음자리표

**Comment by hyunjung-choi** [comment]
높은음자리표

**Comment by sunyeongan** [comment]
16분음표

**Comment by zion830** [comment]
세미콜론

**Comment by hyunjung-choi** [comment, marked as answer]
2분음표

**Comment by DahyeonWoo** [comment]
2분음표
